### PR TITLE
Support to Get IKE Security Associations for VPN Site Link Connections

### PR DESCRIPTION
**Pull request by abhi7860**
## Description
Customers can now fetch IKE Security Associations for VPN Site Link Connections

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/843
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network/help/Get-AzVpnSiteLinkConnectionIkeSa.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
```
Comment (VeryEarly) [inline src/Network/Network/help/Get-AzVpnSiteLinkConnectionIkeSa.md:4]: please complete the online version

Comment (abhi7860) [reply src/Network/Network/help/Get-AzVpnSiteLinkConnectionIkeSa.md]: Done

File: src/Network/Network.Test/Network.Test.csproj [hunk 2]
```diff
@@ -14,11 +14,11 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.3.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="201.359.57612" />
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.5" />
     <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="3.1.0-preview.2" />
     <PackageReference Include="Microsoft.Azure.Insights" Version="0.16.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="39.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="43.0.0-preview" />
```
Comment (VeryEarly) [inline src/Network/Network.Test/Network.Test.csproj:21]: this might potentially have impact on other teams working on this branch

Comment (abhi7860) [reply src/Network/Network.Test/Network.Test.csproj:21]: The Swagger was checked-in in the network-november-release. Until the official SDK is generated, the tests don't pass, right? I plan on reverting those files once that has been done if that's alright. Or I can revert it now?

Comment (abhi7860) [reply src/Network/Network.Test/Network.Test.csproj:21]: What would you recommend? @VeryEarly

Comment (VeryEarly) [reply src/Network/Network.Test/Network.Test.csproj:21]: > What would you recommend? @VeryEarly
the nework SDK is fine, what I mean is the compute SDK.

It looks like a major version upgrade and it's possible that other test cases are not compatible with it.

Comment (VeryEarly) [reply src/Network/Network.Test/Network.Test.csproj:21]: Anyway let's run the CI and see.

**Comment by VeryEarly** [comment]
Hi @abhi7860 ,

There are several other test.csproj need to be updated with the local feed SDK, please see below PR as reference:
https://github.com/Azure/azure-powershell/pull/13905/files

**Review by VeryEarly — COMMENTED** [review]
there are cred found in test, please suppress them all.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**